lib/PuppeteerSharp/PageCoverage/CSSCoverage.cs

```diff
@@ -57,10 +65,15 @@ internal async Task<CoverageEntry[]> StopAsync()
             _enabled = false;
 
             var trackingResponse = await _client.SendAsync<CSSStopRuleUsageTrackingResponse>("CSS.stopRuleUsageTracking").ConfigureAwait(false);
+
```

Comment:
Fixes what I think is a pre-existing issue where the styleSheetAdded signal isn't necessarily received by the time the code reaches this point. I think this is pre-existing b/c if you inject a `await Task.Delay(10)` inside of `OnStyleSheetAddedAsync`  (i.e. simulates a slow response returned by Chromium), then the corresponding unit tests for this class fail. My async changes from this PR seem to exacerbate this issue.

The DeferredTaskQueue and calling DrainAsync() before gathering coverage results are I think what fixes this issue.